internal/super/flix.go

```diff
@@ -49,18 +49,44 @@ type flixFlags struct {
 	GasLimit    uint64   `default:"1000" flag:"gas-limit" info:"transaction gas limit"`
 }
 
+type flixResult struct {
+	flixQuery string
+	result    string
+}
+
 var flags = flixFlags{}
+var FlixCmd = &cobra.Command{
+	Use:              "flix",
+	Short:            "execute, bindings",
+	TraverseChildren: true,
+	GroupID:          "tools",
+}
 
-var FlixCommand = &command.Command{
+var executeCommand = &command.Command{
 	Cmd: &cobra.Command{
-		Use:     "flix <id | name | path>",
-		Short:   "Execute FLIX template with a given id, name, or local filename",
-		Example: "flow flix multiply 2 3",
-		Args:    cobra.ArbitraryArgs,
-		GroupID: "super",
+		Use:     "execute <id | name | path>",
+		Short:   "execute FLIX template with a given id, name, or local filename",
+		Example: "flow flix execute transfer-flow 1 0x123456789",
+		Args:    cobra.MinimumNArgs(1),
 	},
 	Flags: &flags,
-	RunS:  execute,
+	RunS:  executeCmd,
+}
+
+var bindingCommand = &command.Command{
+	Cmd: &cobra.Command{
+		Use:     "bindings <id | name | path>",
```

Comment:
bindings is not a term i like here, but not sure if generate is a lot better and generate-client is very long.

Reply:
Yeah I don't love`bindings` either here tbh

Reply:
what would be a better word? "integrate" or "bind" or "wrap" or "gen-wrap" or "gen-bind", "craft" "conjure", ...

Reply:
good suggestion "package" good suggestion @chasefleming